content/english/anti-vegan/trapped-on-an-island-with-a-pig.md

```diff
@@ -0,0 +1,129 @@
+
+> If it's just you and a pig on a desert island surely you would eat the pig so you could survive yourself? Then you wouldn't be vegan anymore would you?" *(snigger)*.  
+> Followed by: So if you (the vegan) choose to eat the pig, then that's no different from me eating pork when I choose.
+
+This line of reasoning is both fallacious (as discussed shortly) and shows a misunderstanding of the scope and nature of the thought experiment:
+
+## The thought experiment
+
+The following is a more complete version of the thought experiment:
+
+>A ship sinks and there are only two survivors. You and the other survivor have washed up on a small rocky island. You know when the next boat will pass and will be able to signal help. However, you will not survive till then unless you kill and eat your companion.  
+
+Consider that the other survivor is: a child, an adult; your pet dog; a pig.  Work through the thought experiment 4 times. Each time replace the other survivor with one of the options from the list. 
+
+- Do you act the same way on each occasion? 
+- Why/Why not?
+
+*(When rescued it will be assumed the other washed up dead. In the case where the other was a human you will never be charged with their murder)*
+
+### Don’t sidestep the issue!
+
+The important thing about *thought experiments* is that you *think about them*. Don’t rush at justifying an answer; it’s more important to ponder the moral implications.
+
+Some of the sidesteps that should be avoided:
+
+- I’ll eat what the pig’s eating.  *No*, there is absolutely no other food source. 
+- What if the other tried to eat you, so you had to kill them in self-defence?
+- While you wait for rescue, befriend the pig who would then provide company much like your pet cat or dog would? This might deflect the argument on to another interesting discussion but it still sidesteps this important thought experiment. You will *not* survive until rescued unless you kill the other survivor.
+- This couldn’t happen because… *(numerous reasons are presented)*. Yes, the thought experiment presents an improbable scenario but one that is logically possible. However, it really does not matter if the event could occur. Thought experiments are artificial constructs designed to test specific moral principles; you might argue that all thought experiments are invalid but it is far more interesting to engage and consider the implications.
+- You can’t kill the humans; that’s murder!  (Perhaps, but let’s not confuse legality with morality. *Is* the action morally justified? *…and remember you will never be charged with murder.*)
+
+### What’s the answer?
+
+The temptation with thought experiments is to rush to an answer and then be forced to turn cartwheels in an attempt to justify that answer. Consider: 
+
+- Would you kill the animal but never the child? Why? Just because they are human, but without a specific reason or via some claim to human superiority? 
```

Comment:
> Just because they are human, but without a specific reason or via some claim to human superiority? 

This is a little hard to parse. Consider rewording?

Reply:
Would you kill the animal but not a child? Is that *just* because they are human? Are you making a claim of [human superiority]({{< ref "/anti-vegan/humans-are-superior" >}})?

Now just to add citations